core/audio_processor.py

```
@@ -0,0 +1,175 @@
+    async def _convert_to_wav(self, input_path: str, output_path: str) -> None:
+        """Convert audio file to WAV format using FFmpeg."""
+        try:
+            cmd = [
+                "ffmpeg", "-i", input_path,
+                "-ar", str(self._target_sample_rate),
+                "-ac", str(self._target_channels),
+                "-y",  # Overwrite output file
+                output_path
+            ]
+            
+            result = subprocess.run(
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)